source/task_manager/crawl_tasks.py

```diff
@@ -70,39 +72,54 @@ def nutch(self, crawl, rounds=1, *args, **kwargs):
     job_client = nutch_client.Jobs(self.crawl.name)
 
     rest_crawl = nutch_client.Crawl(seed, jobClient=job_client, rounds=self.crawl.rounds_left)
+    nutch_crawl(self.crawl, rest_crawl, url_trails)
```

Comment:
The `save` call can't be here, since the `self.crawl` object in this namespace will be stale after the function return.  

Another option would be to return the crawl_model.